### PR TITLE
Jan week 4 launch

### PR DESCRIPTION
**Pull request by benjaminha14**
https://muse-place-fz1eocwnv.vercel.app/media_seauxcray
https://muse-place-q13zyizof.vercel.app/ozwvld
https://muse-place-fz1eocwnv.vercel.app/abieyuwa
https://muse-place-fz1eocwnv.vercel.app/niajseban
https://muse-place-fz1eocwnv.vercel.app/yunggleeofficial

### CODE REVIEW

File: src/pages/abieyuwa.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import dynamic from "next/dynamic";
+import { GothamProps } from "../themes/Gotham";
+
+const Gotham = dynamic(import("scenes/Gotham"), { ssr: false });
+const ARTIST = {
+  name: "AbieyuwaLD",
+  socials: ["https://www.instagram.com/ozwvld/", "https://vimeo.com/399998090"],
```
Comment (alex-shortt) [inline src/pages/abieyuwa.tsx:9]: wrong links

File: src/pages/abieyuwa.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import dynamic from "next/dynamic";
+import { GothamProps } from "../themes/Gotham";
+
+const Gotham = dynamic(import("scenes/Gotham"), { ssr: false });
+const ARTIST = {
+  name: "AbieyuwaLD",
```
Comment (alex-shortt) [inline src/pages/abieyuwa.tsx:8]: extra LD?

File: src/scenes/Gotham/Abieyuwa/index.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { useCallback, useMemo, useRef, useState } from "react";
+import { MeshStandardMaterial } from "three";
+import { Image, Floating, Interactable } from "spacesvr";
+import { Group } from "three";
+import { useFrame } from "react-three-fiber";
+import CrazyMaterial from "themes/Gotham/shaders/crazy";
+import Walls from "./Walls";
+import Content from "./Content";
```
Comment (alex-shortt) [inline src/scenes/Gotham/Abieyuwa/index.tsx:8]: delete this file if it's unused

File: src/scenes/Gotham/Abieyuwa/Walls.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import { Material } from "three";
+import { useFrame } from "react-three-fiber";
+import { useEffect, useRef } from "react";
+import { Group } from "three";
+
+type PedestalProps = {
+  crazyMaterial: Material;
+  drugTaken: boolean;
+};
```
Comment (alex-shortt) [inline src/scenes/Gotham/Abieyuwa/Walls.tsx:9]: delete this file if it's unused

File: src/scenes/Gotham/Abieyuwa/Content.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
+import { Video } from "spacesvr";
+import { useFrame } from "react-three-fiber";
+import { useRef } from "react";
+import { Group } from "three";
+
+type ContentProps = {
+  drugTaken: boolean;
+};
```
Comment (alex-shortt) [inline src/scenes/Gotham/Abieyuwa/Content.tsx:8]: delete this file if it's unused

**Review by alex-shortt — CHANGES_REQUESTED** [review]

File: src/pages/abieyuwa.tsx (hunk 1 above)
Comment (benjaminha14) [reply src/pages/abieyuwa.tsx]: damn epic fail

File: src/scenes/Gotham/Abieyuwa/index.tsx (hunk 3 above)
Comment (benjaminha14) [reply src/scenes/Gotham/Abieyuwa/index.tsx]: shoot yeah i wanted to replace the walls and add a video she had but she didnt send it over so i removed the entire folder

**Review by alex-shortt — APPROVED** [review]

**Pull request merged**